assets.md

```diff
@@ -40,6 +41,12 @@ This file is intended to list all the assets controlled by OpenTelemetry.
 
 <!-- tocstop -->
 
+## GitHub organization
+
+Link: https://github.com/open-telemetry
+
+- Admins: [@open-telemetry/owners](https://github.com/orgs/open-telemetry/teams/owners
```

Comment:
Do we want to call this "open-telemetry/owners" or "open-telemetry/admins"? (I saw that Kubernetes use "owners")

Reply:
github calls them owners (https://github.com/orgs/open-telemetry/people?query=role%3Aowner), but I agree that admins might be clearer for community members when they want to ping that group

Reply:
will change to admins